example/pom.xml

```diff
@@ -304,4 +303,44 @@ Also contains scripts for deploying a Hono instance to a Docker Swarm, a Kuberne
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>build-docker-image</id>
+      <activation>
+        <property>
+          <name>docker.host</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <configuration>
+              <images>
+                <image>
+                  <name>${docker.image.org-name}/hono-grafana:%v</name>
```

Comment:
we do not want to build our own image for grafana but instead want to use the standard image ...

Reply:
Well the standard image suffers from a few issues:
*  It still has permission problems so you still require "root" permissions, which may not be possible to have in some OpenShift cluster setups. Thus the S2I based deployment already used its own Grafana build.
* The official image doesn't seem to provide an entrypoint for pre-provisioning
* From what I know the standard image isn't refreshed, so you will have stale images

All of this, in combination with the fact that the build for this image is super simple, I think it makes sense to provide a custom version, as it really offers a benefit over the official image.

What would speak against a custom image?

Reply:
That we need to maintain it and it is *not* in Hono's scope.

I understand your pain point regarding OpenShift deployment but frankly, I do not see why the Hono project should be responsible for fixing this problem. Why don't you try to get your fixes incorporated into the *official* Grafana image instead?